src/Microsoft.Framework.Caching.Memory/EntryLink.cs

```diff
@@ -30,5 +44,14 @@ public void SetAbsoluteExpiration(DateTimeOffset absoluteExpiration)
                 AbsoluteExpiration = absoluteExpiration;
             }
```

Comment:
IEntryLink.SetAbsoluteExpiration probably needs a new name like `AddAbsoluteExpiration` that indicates it's not strictly an overwrite.